ts/components/conversation/message/message-content/MessageContent.tsx

```diff
@@ -202,61 +144,36 @@ export const MessageContent = (props: Props) => {
         threshold={0}
         rootMargin="500px 0px 500px 0px"
         triggerOnce={false}
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          gap: 'var(--margins-xs)',
+        }}
       >
         <IsMessageVisibleContext.Provider value={isMessageVisible}>
-          {!isDeleted && (
-            <>
-              <MessageQuote messageId={props.messageId} />
-              <MessageAttachment
-                messageId={props.messageId}
-                imageBroken={imageBroken}
-                handleImageError={handleImageError}
-              />
-            </>
-          )}
           {hasContentAfterAttachmentAndQuote ? (
```

Comment:
I think you can use `&&` here and not `?` and then you don't need to return null.

Reply:
:+1: